api/src/paths/project/{projectId}/attachments/{attachmentId}/delete.ts

```diff
@@ -102,39 +95,3 @@ export function deleteAttachment(): RequestHandler {
     }
   };
 }
-
-const unsecureProjectAttachmentRecord = async (securityToken: any, connection: IDBConnection): Promise<void> => {
```

Comment:
not needed?

Reply:
No use of this according to the requirements